src/org.hdfgroup.object/hdf/object/h5/H5ReferenceType.java

```diff
@@ -286,6 +256,39 @@ public final boolean isInited() {
         return inited;
     }
 
+    //    /**
+//     * Returns the size of the datatype in bytes. For example, for a 32-bit
+//     * integer, the size is 4 (bytes).
+//     *
+//     * @return the size of the datatype.
+//     */
+//    @Override
+//    public long getDatatypeSize() {
```

Comment:
Remove or keep?

Reply:
Not sure, I think I programmed around the 1.12 reference size. So I will remove.